test/specs/core.controller.tests.js

```diff
@@ -1485,8 +1485,8 @@ describe('Chart', function() {
         'afterDataLimits',
         'beforeBuildTicks',
         'afterBuildTicks',
-        'beforeBuildTicks', // y-axis re-fit
-        'afterBuildTicks',
+        // 'beforeBuildTicks', // y-axis re-fit
```

Comment:
Intended?

Reply:
Oh, forgot about this. I think this was actually x-axis re-fit, and is not required anymore for some reason.